Software/Frontend/ws/src/main/java/hr/foi/air/baufind/ws/response/LoginResponse.kt

```diff
@@ -0,0 +1,7 @@
+package hr.foi.air.baufind.ws.response
+
+data class LoginResponse(
+    val success : String,
```

Comment:
Ovo sve dobro izgleda. Jedino što predlažem je da paziš da odgovarajući tipovi budu _nullable_, ovisno kakvi se vraćaju od API-ja. Tako bi u klasama LoginResponse i RegistrationResponse trebao staviti elemente da su "String?".